### ISSUE TITLE
Needs a CONTRIBUTING.md file

### ISSUE THREAD

**Issue by iglpdc** [swcarpentry/git-novice#57]
I'll add one once a pending PR to this file in lesson-template is merged.

**Comment by gvwilson** [comment]
Which PR needs merged in `lesson-template`?

**Comment by iglpdc** [comment]
@gvwilson you just merged it :-) https://github.com/swcarpentry/lesson-template/pull/183

**Comment by jdblischak** [comment]
@iglpdc, you can merge the CONTRIBUTING.md file into this repo by pulling the changes in the `core` branch of lesson-template. I'm not aware of this process being documented anywhere. Here's how I normally do it:

```
git remote add lesson-template git@github.com:swcarpentry/lesson-template.git
git checkout -b merge-core
git pull lesson-template core
# Fix any merge conflicts
git checkout gh-pages
git merge merge-core
git push swc gh-pages # swc is the swcarpentry repo, alternatively you can push to your fork and then submit a PR
```

**Comment by iglpdc** [comment]
Awesome, @jdblischak! Done. Thanks for the help. 

Fixed by b7dcedcfd7fff63729ed9a664f728d70f2e34e29.
